integrations/object_store/src/store.rs

```diff
@@ -412,6 +417,74 @@ impl ObjectStore for OpendalStore {
     }
 }
 
+/// `MultipartUpload`'s impl based on `Writer` in opendal
+struct OpendalUpload {
```

Comment:
How about following the same naming pattern like `object_store`: `OpendalMultiPartUpload`?

Reply:
Done